samples/bluetooth/hci_rpmsg/src/main.c

```diff
@@ -33,13 +38,19 @@ static K_THREAD_STACK_DEFINE(tx_thread_stack, CONFIG_BT_HCI_TX_STACK_SIZE);
 static struct k_thread tx_thread_data;
 static K_FIFO_DEFINE(tx_queue);
 static K_SEM_DEFINE(ipc_bound_sem, 0, 1);
+#if defined(CONFIG_BT_CTLR_ASSERT_HANDLER) || defined(CONFIG_BT_HCI_VS_FATAL_ERROR)
+static bool ipc_ept_ready;
```

Comment:
And since this is a sample, may be worth to add a comment about this too (we have received feedback on Discord that many samples are using acronyms and abbreviations that aren't obvious to many new users).

Reply:
I'll add a comment. Regarding the name, it follows convention from this source file e.g. 'hci_ept_bound`, `hci_ept_recv`.
It looks like we have two conventions for end point naming.